assets/pages/regular-contributions/components/contributionsGuestCheckoutContainer.jsx

```diff
@@ -0,0 +1,82 @@
+// @flow
+
+// ----- Imports ----- //
+
+import { connect } from 'react-redux';
+import type { Dispatch } from 'redux';
+import { type UserFormFieldAttribute } from 'helpers/checkoutForm/checkoutForm';
+import ContributionsGuestCheckout from './contributionsGuestCheckout';
+import { type State } from '../regularContributionsReducer';
+import { getFormFields } from '../helpers/checkoutForm/checkoutFormFieldsSelector';
+import {
+  setEmailShouldValidate,
+  setFirstNameShouldValidate,
+  setLastNameShouldValidate,
+  setStage,
+} from '../helpers/checkoutForm/checkoutFormActions';
+import { type Action as CheckoutAction } from '../helpers/checkoutForm/checkoutFormActions';
+
+
+// ----- Functions ----- //
+
+const setShouldValidateFunctions = (dispatch: Dispatch<CheckoutAction>) => [
```

Comment:
Might be clearer to have:
```javascript
const setFirstNameShouldValidate = [
  setFirstNameShouldValidate,
  setLastNameShouldValidate,
  setEmailShouldValidate,
]
```

And then if the form isn't valid:
```javascript
setShouldValidateFunctions.forEach(f => dispatch(f()))
```

Reply:
love this